src/schema/schema_drop.c

```diff
@@ -174,31 +174,84 @@ __drop_table(WT_SESSION_IMPL *session, const char *uri, const char *cfg[])
  *     Drop a tiered store.
  */
 static int
-__drop_tiered(WT_SESSION_IMPL *session, const char *uri, const char *cfg[])
+__drop_tiered(WT_SESSION_IMPL *session, const char *uri, bool force, const char *cfg[])
 {
+    WT_CONFIG_ITEM cval;
     WT_DATA_HANDLE *tier;
     WT_DECL_RET;
     WT_TIERED *tiered;
     u_int i;
+    const char *name;
+    bool remove_files;
+
+    WT_RET(__wt_config_gets(session, cfg, "remove_files", &cval));
+    remove_files = cval.val != 0;
 
+    name = NULL;
+    WT_UNUSED(cfg);
```

Comment:
Does the compiler require `WT_UNUSED` here? We used `cfg` a few lines up.

Reply:
Good catch. Removing.